src/inbox/mod_inbox.erl

```diff
@@ -114,7 +118,8 @@ supported_features() ->
 config_spec() ->
     Markers = mongoose_chat_markers:chat_marker_names(),
     #section{
-        items = #{<<"backend">> => #option{type = atom, validate = {enum, [rdbms]}},
+        items = #{<<"backend">> => #option{type = atom, validate = {enum, [rdbms, rdbms_async]}},
+                  <<"async_writer">> => async_config_items(),
```

Comment:
Minor: I would change the name to something like `async_config_spec`, because it is not returning items anymore.